src/EditorFeatures/Test2/FindReferences/FindReferencesTests.ParameterSymbol.vb

```diff
@@ -180,10 +180,11 @@ partial class program
     <Project Language="Visual Basic" CommonReferences="true">
         <Document>
         Class C                
-           partial sub PM(x as Integer, y as Integer)
```

Comment:
Fixing this test as it was just bogus.  In VB, only one of the parts of a partial method can have the 'partial' modifier.